token/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@solana/spl-token",
-  "version": "0.0.13",
+  "version": "0.0.14",
```

Comment:
I think we could move this package to 0.1.0, since there is a lot of projects depending on it

Reply:
@dummytester123 can you bump to 0.1.0 like bartosz suggested?

Reply:
oh, totally missed that.